### PR TITLE
Lottycoupat project meeting notes

### PR DESCRIPTION
**Pull request by lottycoupat**
Added new folder to add meeting notes from project meetings with Kirstie. 

Please review notes and spelling.

### CODE REVIEW

**Comment by lottycoupat** [comment]
@GeorgiaHCA This includes notes from both meetings now:

Project meeting 25/05/21
Development meeting 27/05/21

File: platform-designs/user-flow/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# User Flow 
+
+To illustrate how the platform will work and how a user would navigate through it, we have created a user flow. 
+
+This can be seen below:
+
+![](../platform-designs/user-flow/user-flow.png)
```
Comment (GeorgiaHCA) [inline platform-designs/user-flow/README.md:7]: This looks the right formatting but I'm not 100% sure the image will render. Where did you store it?

**Review by GeorgiaHCA — APPROVED** [review]
Really graet work here @lottycoupat thank you!

**Comment by GeorgiaHCA** [comment]
https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/images - here's the folder to store images where we also need to tidy up!

File: platform-designs/user-flow/README.md (hunk 1 above)
Comment (lottycoupat) [reply platform-designs/user-flow/README.md:7]: @GeorgiaHCA I uploaded in the user flow folder...

**Pull request merged**